### DISCUSSION TITLE
Hide Realm

### DISCUSSION THREAD

**Discussion by MeusRex** [MeusRex/pf2e-chex#5] [Ideas]
Sometimes it is useful to hide a realm. E.g. the GM might not want to players to know it exists for now.
In the config there should be a checkbox for each realm.
If the realm is invisible, its borders should not be visible to players and it should not appear in the HUD.

- [ ] Option
- [ ] HUD
- [ ] Overlay
- [ ] Coat of arms

**Comment by federicoacker** [comment]
Might also be cool to display a "realm image" in the owner field over highlighted hexes

**Reply by MeusRex** [reply]
Hmm Coat of Arms, I'll add it to the list
